frontend/lib/src/hostComm/HostCommunicationManager.tsx

```diff
@@ -110,18 +110,18 @@ export default class HostCommunicationManager {
 
   /**
    * Function to set the response body received from hitting the Streamlit
-   * server's /_stcore/allowed-message-origins endpoint. The response contains
+   * server's /_stcore/host-config endpoint. The response contains
    *   - allowedOrigins: A list of origins that we're allowed to receive
    *     cross-iframe messages from via the browser's window.postMessage API.
    *   - useExternalAuthToken: Whether to wait until we've received a
    *     SET_AUTH_TOKEN message before resolving deferredAuthToken.promise. The
    *     WebsocketConnection class waits for this promise to resolve before
    *     attempting to establish a connection with the Streamlit server.
    */
-  public setAllowedOriginsResp = ({
+  public setHostConfigResp = ({
     allowedOrigins,
     useExternalAuthToken,
```

Comment:
nit: I would probably make this independent of the host config. So, it's just a method that configures the options we actually need for HostCommunicationManager. And in App.tsx we are getting the properties from host config and calling this function with it. 

```suggestion
  public setAllowedOrigins = (
    allowedOrigins: string[],
    useExternalAuthToken: boolean
 )
```

Reply:
Makes sense. Thanks